feat(Peer Chat): Show chat members at top of Peer Chat

geoffreykwan opened the pull request:
## Changes
We now show chat member names at the top of the Peer Chat

## Test
- Make sure chat member names show up at the top of the Peer Chat in the student view
- Make sure chat member names show up at the top of the Peer Chat in the grading tool

Closes #511

geoffreykwan committed:
feat(Peer Chat): Show chat members at top of Peer Chat

#511

breity committed:
feat(Peer Chat): Show names at top of PeerChatBox
#511

breity reviewed (changes requested):
Great, looks good. I updated to show the names at the top of the chat box instead of in a card above.

One issue: The list of workgroups differs between student view and teacher grading view. In grading, only users that have posted to the chat are listed. We should show all group members, like in VLE. Grading view also includes the teacher in the list if teacher has posted. We shouldn't list the teacher.

geoffreykwan committed:
fix(Peer Chat): Show all peer chat members and not the teacher at top

Previously we were not showing student members that had not sent a
response and we were also showing the teacher.

#511

geoffreykwan committed:
Merge branch 'issue-370-create-peer-chat-component' into issue-511-show-peer-chat-members

breity reviewed (approved):
Looks good. 👍
